static/elements/chromedash-new-feature-list.js

```diff
@@ -0,0 +1,86 @@
+import {LitElement, css, html} from 'lit';
+import './chromedash-approvals-dialog';
+import './chromedash-feature-table';
+import SHARED_STYLES from '../css/shared.css';
+
+
+class ChromedashNewFeatureList extends LitElement {
+  static get properties() {
+    return {
+      signedInUser: {type: String},
+      canEdit: {type: Boolean},
+      canApprove: {type: Boolean},
+      loginUrl: {type: String},
+      starredFeatures: {type: Object}, // will contain a set of starred features
+    };
+  }
+
+  constructor() {
+    super();
+    this.signedInUser = ''; // email address
+    this.starredFeatures = new Set();
+    this.canEdit = false;
+    this.canApprove = false;
+  }
+
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      css`
+      `];
+  }
+
+  // Handles the Star-Toggle event fired by any one of the child components
+  handleStarToggle(e) {
+    const newStarredFeatures = new Set(this.starredFeatures);
+    window.csClient.setStar(e.detail.featureId, e.detail.doStar)
+      .then(() => {
+        if (e.detail.doStar) {
+          newStarredFeatures.add(e.detail.featureId);
+        } else {
+          newStarredFeatures.delete(e.detail.featureId);
+        }
+        this.starredFeatures = newStarredFeatures;
+      })
+      .catch(() => {
+        alert('Unable to star the Feature. Please Try Again.');
```

Comment:
Do we want a browser alert here, or is this something that normally never gets used?

Reply:
It should almost never happen so I went with the simplest handler.  We also have a "toast" element, so I suppose that I could have used that.  I'll keep that in mind if eventually improve our error handing.